CHANGES.md

```diff
@@ -1,6 +1,10 @@
+# 1.2.6
+
+- [FIXED] ComplianceFetcher.session can now be reset.
+
 # 1.2.5
 
-- [FIXED] Credentials section bug affecting the Slack notifier.
+- [FIXED] Credentials section bug affecting the Slack notifier is squashed.
```

Comment:
"is squashed" seems unnecessary given the `[FIXED]`?

Reply:
I specifically added it because to me `[FIXED]` is the tag and not meant to be part of description.  So I didn't want to say "[FIXED] Fixed credentials section bug..." nor did I just want to say "[FIXED] Credentials section bug...".  IDK, it makes sense to me :).